windows/ReactNativeWebView/ReactWebView.cpp

```diff
@@ -132,6 +139,16 @@ namespace winrt::ReactNativeWebView::implementation {
         PostMessage(winrt::hstring(args.Value()));
     }
 
+    void ReactWebView::OnPremissionRequested(winrt::WebView const& webView, winrt::WebViewPermissionRequestedEventArgs const& args)
+    {
+      if (args.PermissionRequest().PermissionType() == WebViewPermissionType::ImmersiveView) {
+        // WebGL permission request
+        // https://docs.microsoft.com/en-us/microsoft-edge/webvr/webvr-in-webview
+        args.PermissionRequest().Allow();
```

Comment:
"When you allow the requested permission, subsequent calls to Allow, Deny, or Defer the permission request result in an error."
(same with Deny)

Does that mean you have to cache the last result so that you don't make a new call to Allow/Deny which can result in an exception being thrown?

Reply:
I does not have any problem with this implementation. All examples in the internet what i have found are implemented  as quite same. Like this https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/web-view